### PR TITLE
Fix W3C errors

### PR DESCRIPTION
**Pull request by l33d4n**
Errors list: https://validator.w3.org/nu/?doc=https%3A%2F%2Fgrinplusplus.github.io%2F

## Before:

image

## After:

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -360,15 +360,14 @@ <h2 class="featurette-heading">
...
-        <a name="contributing"><h1 class="text-center">Contributing</h1></a>
+        <h1 class="text-center">Contributing</h1>
       </div>
 
       <div class="row">
         <div class="col-lg-4">
           <img src="/images/donate-icon.png" alt="bitcoin" /><br /><br />
           <p>
             <a
-              id="windows-download"
```
Comment (davidtavarez) [inline index.html:371]: I'm afraid this will break the links.

File: index.html [hunk 2]
```diff
@@ -396,7 +394,6 @@ <h2 class="featurette-heading">
           <img src="/images/bug-icon.png" alt="macOS" /><br /><br />
           <p>
             <a
-              id="macos-download"
```
Comment (davidtavarez) [inline index.html:399]: the `id` is used to replace the `href` with the proper link.

**Review by davidtavarez — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (l33d4n) [reply index.html:399]: The changes are for the area in the image, these are static links with the same duplicate ID as the download area. I checked again, there is no effect on the links.

image

Comment (davidtavarez) [reply index.html:399]: sorry, you're right!

**Review by davidtavarez — APPROVED** [review]

**Pull request merged**